windows/jitsi-meet/index.html

```diff
@@ -18,6 +18,7 @@
         </style>
     </head>
     <body>
+      <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
As far as I tried, RTCPeerConnection was undefined without the adapter. I guess lib-jitsi-meet's adapter is contained within the iframe, so the Electron browser does not have access to it.